src/python_routing_v02/troute/routing/fast_reach/mc_reach.pyx

```diff
@@ -1151,6 +1151,7 @@ cpdef object compute_network_structured(
                                    array('l',upstream_ids),
                                    wbody_parameters[wbody_index])
                     )
+                wbody_index += 1
```

Comment:
@jdmattern-noaa what is the purpose of this increment here?

Reply:
This is a relic from the previous implementation for indexing reservoirs before the binary_find was added that I accidentally left in. I will remove it. 
https://github.com/NOAA-OWP/t-route/blob/7daa4ffc2db1996e384c3cfbb03eba0424b8e279/src/python_routing_v02/troute/routing/fast_reach/mc_reach.pyx#L1091